src/test/java/org/graylog/integrations/dataadapters/GreyNoiseDataAdapterTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.integrations.dataadapters;
+
+import okhttp3.MediaType;
+import okhttp3.Protocol;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.assertj.core.api.Assertions;
+import org.graylog.integrations.dataadpaters.GreyNoiseDataAdapter;
+import org.graylog2.plugin.lookup.LookupResult;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class GreyNoiseDataAdapterTest {
```

Comment:
I think once you move the `OkHttpClient` into the constructor, you can mock it out and that will allow you to write a couple more test cases such as:

* Verifying that the expected URL is being used to make the call
* Verifying the behavior when the HTTP request isn't successful